packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -192,10 +192,6 @@ governing permissions and limitations under the License.
   justify-content: flex-end;
   /* this padding should be safe as button group is always end aligned */
   padding-inline-start: var(--spectrum-dialog-gap-size);
-
-  &.spectrum-Dialog-buttonGroup--noFooter {
```

Comment:
This messes up the width of the buttons when vertical. @snowystinger please halp